ha/plugin/motr/motr.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+"""
+ ****************************************************************************
+ Description:       Motr plugin to provide motr utility and interfaces.
+ ****************************************************************************
+"""
+
+import json
+
+from cortx.utils.log import Log
+from ha import const
+
+class Motr:
```

Comment:
Why Motr and S3 plugin for just getting Fids? Instead of, you should have a FidManager class which will get the fids for different components.

Reply:
Currently seen fid mapping in file have different schema for s3 and motr
so even we used FidManager we need two different method/class to parse and get fid for s3/motr.

Reply:
Create FidManager class but still keeping plugin as s3 and motr have different logic to get fid.

resource -> FidManager -> plugin_fid